packages/newspringchurchapp/src/ui/WebBrowser/Browser.ios.js

```diff
@@ -2,16 +2,23 @@ import { Linking } from 'react-native';
 import SafariView from 'react-native-safari-view';
 
 const Browser = {
-  openURL: async (url, options) => {
-    try {
-      await SafariView.isAvailable();
-      SafariView.show({
-        url,
-        ...options,
-      });
-    } catch (e) {
+  openURL: async (url, { externalBrowser, ...options }) => {
+    const openWithSafari = () => {
       if (!Linking.canOpenURL(url)) throw new Error('URL not supported');
       Linking.openURL(url);
+    };
+    if (!externalBrowser) {
```

Comment:
I feel like this is backwards. If I want to open an external browser then this should be true.

Reply:
yep :)